saleor/graphql/invoice/mutations.py

```diff
@@ -173,17 +174,19 @@ def perform_mutation(cls, _root, info, **data):
         invoice.order = order
         invoice.status = JobStatus.SUCCESS
         invoice.save()
+        app = load_app(info.context)
+        # app = AppByTokenLoader(info.context).load(info.context.auth_token)
```

Comment:
To remove I suppose. 👮‍♀️

Reply:
Should be marked as `# TODO` or `# FIXME` as I'm working on this part for future PR.

Reply:
Nah, it uses `load_app` so the comment should be removed. Thanks!